(DOCSP-30466) Adds RNs for 1.7.2 (#145)
mongodb/docs-atlas-cli: source/atlas-cli-changelog.txt

   :depth: 1
   :class: singlecol

.. _atlas-cli-1.7.2:

{+atlas-cli+} 1.7.2
-------------------

Released 7 June 2023

> - Fixes a bug that caused the :ref:`atlas-clusters-search-indexes-create` command to 

Reviewer: ```suggestion
- Fixes an issue that caused the :ref:`atlas-clusters-search-indexes-create` command to 
```

Reviewer: To align with second point

Author: Good point! Changed.

  incorrectly handle a multikey index.
- Fixes an issue with the :ref:`atlas-performanceAdvisor-slowQueryLogs` command where the
  output didn't display correctly in tabular form.

> - Adds the ``--tag`` flag to the commands :ref:`atlas-clusters-create`, :ref:`atlas-clusters-update`,

Author: Removed!

> - Adds the ``--tag`` flag to the commands :ref:`atlas-clusters-create`, :ref:`atlas-clusters-update`,

Reviewer 2: Hi @wtrocki Was this recently reverted? I see it appears on the docs page in staging for atlas clusters create so wondering if that needs to change too: https://docs-mongodbcom-staging.corp.mongodb.com/atlas-cli/docsworker-xlarge/DOCSP-30466/command/atlas-clusters-create/#options

> - Adds the ``--tag`` flag to the commands :ref:`atlas-clusters-create`, :ref:`atlas-clusters-update`,

Reviewer: Change is waiting for API to release. Then it be included in the next version.
This is tradef us using atlas-cli-next release. If there is hot fix in the mean time while we wait for feature release we going to have changelog with invalid entries. It should not happen that offten
